### DISCUSSION TITLE
[🌟] Needed crop feature

### DISCUSSION THREAD

**Discussion by rajeshivn** [react-native-image-picker/react-native-image-picker#1682] [Ideas]
### Description

Explain the feature request in as much detail as possible.

### Proposed Solution

What needs to be done to make it happen?

### Test Solution

How would we test & validate the feature?

**Comment by rpander93** [comment]
Note: I recently published an image cropper component for React Native that you can integrate in your pick-an-image-workflow. See https://github.com/rpander93/react-native-image-editor-ui

**Comment by NiharR27** [comment]
+ 1
